### PR TITLE
fix: Handle GitHub and BitBucket Server review approvals same as /approve

### LINKED ISSUE #769: PRs approved with github's review system don't get the 'approved' label
when approving a PR using github's review system, lighthouse won't add the `approved` label.

this used to work on CJXD 9 with prow, but on the same cluster with same (prow) config on CJXD 10 with lighthouse, it doesn't work anymore.

just to be clear: commenting with a `/approve` cmd works.

### PR DESCRIPTION
**Pull request by abayer**
fixes #769

Signed-off-by: Andrew Bayer <andrew.bayer@gmail.com>

### CODE REVIEW

**Comment by abayer** [comment]
Yaaaaaaay - verified this works for GitHub. BitBucket Server still needs a bit of work on the go-scm side, but I'll do that next week.

File: pkg/webhook/webhook_test.go [hunk 1]
```diff
@@ -57,6 +57,28 @@ func (suite *WebhookTestSuite) TestProcessWebhookPR() {
...
 }
 
+func (suite *WebhookTestSuite) TestProcessWebhookPRReview() {
+	t := suite.T()
+
+	webhook := &scm.ReviewHook{
+		Action: scm.ActionSubmitted,
+		Repo:   suite.TestRepo,
+		Review: scm.Review{
+			State: "APPROVED",
```
Comment (vbehar) [inline pkg/webhook/webhook_test.go:67]: quick question: does it also handle the "request changes" state?

Comment (abayer) [reply pkg/webhook/webhook_test.go:67]: Yup! And dismissing a review.

**Pull request merged**